public/assets/js/impact_discover/impact_discover.js

```diff
@@ -0,0 +1,261 @@
+/*
+@licstart  The following is the entire license notice for the
+JavaScript code in this page.
+
+Copyright (C) 2010  Goteo Foundation
+
+The JavaScript code in this page is free software: you can
+redistribute it and/or modify it under the terms of the GNU
+General Public License (GNU GPL) as published by the Free Software
+Foundation, either version 3 of the License, or (at your option)
+any later version.  The code is distributed WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS
+FOR A PARTICULAR PURPOSE.  See the GNU GPL for more details.
+
+As additional permission under GNU GPL version 3 section 7, you
+may distribute non-source (e.g., minimized or compacted) forms of
+that code without the copy of the GNU GPL normally required by
+section 4, provided you include this license notice and a URL
+through which recipients can access the Corresponding Source.
+
+
+@licend  The above is the entire license notice
+for the JavaScript code in this page.
+*/
+
+//reset ODS JSON
+function resetODS() {
+    $.each(odsList.ods, function(key, value){
+        odsList.ods[key]["active"] = false
+    })
+}
+
+// reset ODS select
+function resetODSSelect() {
+    $(".impact-discover-filters select option").remove();
+    $(".impact-discover-filters select").append("<option>Filtra por Objetivos de Desarrollo sostenible</option");
+}
+
+// check if ODS option has selected footprint
+function hasFootprint (ods, footprint) {
+    var footprints = ods.footprints;
+    return footprints.indexOf(footprint)>=0 ? true : false;
+}
+
+// add ODS to ODS select
+function addODSToSelect(ods) {
+    $(".impact-discover-filters select").append('<option data-footprints="'+ods.footprints+'">'+ods.ods+'</option>');
+}
+
+// reset ODS icons
+function resetODSIcons() {
+    $('.impact-discover-projects h1').text("Busca un proyecto por Huellas o ODS");
+    $("#ods-icons .col").html("");
+}
+
+// add ODS to icon list
+function addODSIcon(ods) {
+    $('.impact-discover-projects h1').text("Proyectos que cumplen con los siguientes ODS:");
+    var icon = '<img src="./assets/img/ods/'+ods.id+'.svg" />';
+    var removeIcon = '<a class="close flip" href="#"><i class="icon icon-close"></i></a>';
+    $("#ods-icons .col").append('<div class="odsicon" data-ods="'+ods.id+'">'+icon+removeIcon+'</div>');
+}
+
+// activate ODS on JSON
+function activateODS(ods){
+    $.each(odsList.ods, function(key, value){
+        if (value.id == ods.id) odsList.ods[key]["active"] = true
+    })
+    addODSToSelect(ods);
+}
+
+// remove ODS
+function removeODS(ods){
+    $.each(odsList.ods, function(key, value){
+        if (value.id == ods) odsList.ods[key]["active"] = false
+    })
+    refreshODS();
+}
+
+function isODSActive(ods) {
+    return ods.active;
+}
```

Comment:
@davidbeig is this method really necessary? Couldn't we use ods.active in the other JS code?